spec/requests/vendor_api/post_make_an_offer_spec.rb

```diff
@@ -65,6 +65,39 @@
     end
   end
 
+  describe 'making an offer with conditions with non UTF-8 characters' do
```

Comment:
Should this read 'with non-ASCII characters'?

From the error message, it looks like somewhere along the way the interpreter treats the string as if it were ASCII_8BIT,   calls something like `encode(Encoding::UTF_8)` on it, and then errors because it can't convert the non-ASCII character.